tests/std/tests/P0980R1_constexpr_strings/test.cpp

```diff
@@ -1530,6 +1526,7 @@ _CONSTEXPR20_CONTAINER bool test_iterators() {
         cit = cit2;
     }
 
+#if defined(MSVC_INTERNAL_TESTING) || defined(__EDG__) // TRANSITION, VSO-1270433

```

Comment:
I am obviously too bad at copy & paste, does not feel like I really want to rerun just for that though

Reply:
I believe that was the wrong bug number